modules/tinymce/src/core/test/ts/browser/FormatterCheckTest.ts

```diff
@@ -163,6 +163,63 @@ describe('browser.tinymce.core.FormatterCheckTest', () => {
     assert.isTrue(editor.formatter.match('complex', { color: '#ff00' }), 'Selected element match with variable and function');
   });
 
+  context('using similar', () => {
```

Comment:
This should also probably be this:
```suggestion
  context('match using similar', () => {
```